### DISCUSSION TITLE
Planning for Art Collections in Arctos

### DISCUSSION THREAD

**Discussion by marecaguthrie** [ArctosDB/arctos#4341] [Ideas]
**Arctos vs Core Categories for the Description of Works of Art**

@krgomez and I have been working on a big-picture analysis of best practices for cataloging art collections and how we might be able to be more strategic in discussing ideas and requests for Arctos. We directly compared Arctos to the Getty's Categories for the Description of Works of Art (CDWA). The CDWA is a data content standard for the visual arts and related disciplines that identifies fields, vocabulary sources, and descriptive practices to make information more accessible and comparable for sharing and linking. CDWA includes includes 540 categories but we have focused on the small subset that are considered core -- the minimum information necessary. For more information about the CDWA see the [CDWA one-page overview pdf](https://www.getty.edu/research/tools/vocabularies/vocab_cdwa_flier.pdf) or the more detailed [CDWA 62 page pdf.](https://getty.edu/research/publications/electronic_publications/cdwa/definitions.pdf)

This was an incredibly valuable exercise for us and has left us with a clear list of priorities. We hope that there will be ideas and questions that are valuable for other collections as well. We understand that making Arctos a multidisciplinary database takes time and money and that some things won’t be practical until after the PG move but our hope is to have some of these broader conversations about how we might move forward assuming this is compatible with the mission and goals of Arctos.

**Links to our planning documents:**
1. [Top Arctos Priorities to Address for Art Collections](https://docs.google.com/document/d/1v-rJD-N1SRt5W_ndUWBRobiJ8rPMdzJNMeK12EWcQ8A/edit?usp=sharing) is a shorter document drawn all the work done in planning document below.

2. [18 page planning document](https://docs.google.com/document/d/1hzgPoHetLWSn3-TLLKfnwhUYPzMLjZQbPmNw_VCc97E/edit?usp=sharing) that includes:

- Priorities and Considerations
- Designing for Art Users - Interface Needs
- Link to spreadsheet comparison between Arctos and CDWA
- Summary of Problems and Proposals
- References and Resources

3. [Spreadsheet comparing Getty Categories for the Description of Works of Art (CDWA) with Arctos](https://drive.google.com/file/d/19D7m2nTTser4_YT4MO_OYTdYWDTWibLU/view?usp=sharing)

[The folder on Drive with all planning documents can be found here](https://drive.google.com/drive/folders/15xxQBM2dfNsRKRuHI4EZuYw9-7sbFRh_?usp=sharing)

**Comment by marecaguthrie** [comment]
Neither Karinna or I have any background or experience in databases or coding, but we have tried to figure out what the current conversations in the art museum community are. We think we have identified some of the key players but we will need help interpreting the resources listed below and evaluating how helpful they can be for our discussions.

**The TLDR version:** There seems to be national conversation of how science museums, art museums, other cultural heritage institutions like archives and libraries can all link up their data to make it richer and more discoverable. It seems like CIDOC CRM is at the center of this (?) and that the folks we really need to be watching as far as data relationships go is [Linked Art](https://linked.art/) and that a working draft of their code is [available on GitHub here](https://github.com/linked-art/linked.art).


**_[INTERNATIONAL COMMITTEE FOR DOCUMENTATION CONCEPTUAL REFERENCE MODEL (CIDOC CRM) & LIGHTWEIGHT INFORMATION DESCRIBING OBJECTS (LIDO)](http://www.cidoc-crm.org/)_** 

Because the full CDWA was so large the CDWA Lite was created as an XML schema to describe core records for works of art and material culture based on CDWA and Cataloging Cultural Objects (CCO). The CDWA Lite schema was integrated into the Lightweight Information Describing Objects (LIDO). LIDO is a specific XML based application of CIDOC Conceptual Reference Model (“CIDOC CRM”), a “formal ontology intended to facilitate the integration, mediation and interchange of heterogeneous cultural heritage information”. [The most recent version is available on the CIDOC site](http://www.cidoc-crm.org/versions-of-the-cidoc-crm). I’m not sure I understand the relationship of LIDO to CIDOC CRM but this is [what wikipedia says](https://en.wikipedia.org/wiki/LIDO#Relationship_to_CIDOC_CRM):

> CIDOC CRM is an object-oriented and extensible ontology. It defines concepts and relationships that are necessary for the description of cultural heritage objects. Being an ontology, CIDOC CRM is based on formal logic and graph oriented.
> LIDO, on the other hand, is a specific XML based application of CIDOC CRM. It has been developed for harvesting metadata. Other than CIDOC CRM LIDO is a schema and thereby strictly hierarchic. It adopts aspects of its predecessor, CDWA, but heavily relies on CIDOC CRM's classes, e.g. E5 Event, E39 Actor, E55 Type.

**_[LINKED ART](https://linked.art/)_**

“Linked Art is a [community](https://linked.art/community/) working together to create a shared [model ](https://linked.art/model/)based on Linked Open Data to describe Art”. [Their code (in progress) can be found on GitHub here](https://github.com/linked-art/linked.art). As I understand it they are building off of CIDOC CRM. 

We found this diagram from the presentation on their homepage very useful in thinking about how it compares to Arctos.
![IMG_1885](https://user-images.githubusercontent.com/39135431/73482252-d7c66b00-4349-11ea-9a5a-54881b259db8.jpg)

I’m also copying what seems like critical information below from their [model page](https://linked.art/model/index.html):

> Model Fundamentals
> Following the existing norms of the community, our starting point consists of:
> 
> [CIDOC-CRM](http://www.cidoc-crm.org/) as the core conceptual model, giving an event-based paradigm.
> 
> - We use a streamlined [profile](https://linked.art/model/profile/) of CIDOC-CRM to ensure consistency and comprehension.
> - We also use the RDF implementation of CIDOC-CRM, given our choice of web and linked data as an integration platform.
> 
> The [Getty Vocabularies ](http://vocab.getty.edu/)as core sources of identity for domain-specific terminology
> 
> - Core terminology is part of the application profile documentation directly, when interoperability between systems relies on the selection.
> - Other best practices are discussed in the [vocabularies best practice section](https://linked.art/community/best-practices/vocabularies/).
> 
> [JSON-LD](https://json-ld.org/) as the primary target serialization to ensure ease of use
> 
> - We use a specific [context](https://linked.art/model/jsonld/) designed to be as easy to implement and consume as possible.
> 
> These are then expanded on in order to fulfill shared use cases and common requirements, as they become known.

[**_AMERICAN ART COLLABORATIVE_** ](http://americanartcollaborative.org/)
See “[Ontologies and the CIDOC CRM” presentation on their website](http://americanartcollaborative.org/educational-briefings/ontologies-and-the-cidoc-crm/)

> The American Art Collaborative is dedicated to creating a diverse critical mass of LOD on the Web on the subject of American art by putting the collections of the participating museums in the cloud and tagging this data as LOD. This will exponentially enhance the access, linking, and sharing of information about American art in a way that transcends what is currently possible with structured data. The goals of the initiative are to simplify access to digital information across museum collections for both researchers and the public, to create new opportunities for discovery, research and collaboration, to promote creative reuse of data on American art across numerous applications, and to showcase for museums and other scholarly institutions the inherent possibilities of LOD.

**Comment by mkoo** [comment]
This is incredibly valuable roadmapping and background on art collections- thanks.

I would like to create a similar doc for Archives. This will be a useful model.  Will create a separate issue. Might require a separate repo or other project management organization.

**Comment by dustymc** [comment]
Thanks @marecaguthrie! Just some surface comments for now...

Arctos has shared data via exchange standards (and occasionally not-so-standard formats) since inception, and is built to be connected. I suspect it's technically trivial (if perhaps labor-intensive) to do similar things for other types of data.

I think we'd all embrace more-shared vocabularies. The only "problem" I see with Getty is the scope - it's limited, how do we integrate Getty terms with things that aren't in Getty but which Arctos needs? I think that's near 100% social problem - it's just something we as a community need to figure out.

We will hopefully have funding for a full-fledged API somewhat soon (and an test API sufficient to create specimen search and result should be exposed next week). I'm coming to terms with the idea that we've simply outgrown any single UI; our user base is just too diverse, and our data too deep, for such a thing to be practical. Throwing up an API in front of a collection or group of collections should be in the "technically trivial if perhaps labor intensive" camp very soon.

Things like deeper agent data may be influenced by that - we could add something to Arctos itself (my least-favorite approach by a very wide margin, assuming we trust things like wikidata), pull those data from some API(s) and pipe them through Arctos, or APIs that talk to the Arctos API could also pull from those remote API(s). 

More generally, I firmly believe we should build things only when we can't plug in to existing tools (which often come with existing communities).

There's definitely some weird overlap between "what we call it" (eg, identification) and "what it IS" and/or "how we categorize it" (parts, maybe, and probably some other stuff as well).

Perhaps somewhat related, "materials" is poorly-structured - I'm not sure what the ideal solution is (or how it might line up with reality), but we should be able to do better than allowing an infinite number of attempts at spelling "moose hide" in a free-text field.

There are various ways to link agents to events - I'm not sure what that comment is referencing.

Part condition has always maintained a history, including dates and agents (but pulled from the environment, not user-supplied). Moving that to (part) attributes will provide a place to explicitly include any agent and date.

"Treatment" is clearly part attributes; we're moving towards that in general, and I think "natural history" and "cultural" collections are doing the same things there, just perhaps recording it somewhat differently.

**Comment by marecaguthrie** [comment]
Thanks for such quick responses @mkoo and @dustymc! I think I have a better understanding of what art collections need out of Arctos but I certainly have no idea about the best way to go about it. I am not at all tied to the idea of using the Getty AAT directly as there are manyAlaska-specific things that we would want to use that are not in the Getty AAT. I just want to be planning strategically to standardize the vocabulary as much as possible for the future so that the art part of Arctos can talk to the art part of other databases when the time comes --and I really don't know the best way to do that. Let me know what might be the best way to go about discussing all of this, if a video or phone chat would be easier. Or if we can pay for some of your time so that you can spend some time with this and advise us on how we might proceed.

**Comment by Jegelewicz** [comment]
You guys are really awesome! I need more time to digest everything, especially the CIDOC CRM.

For me, all of this points to the need for strategy and better methods for discussing and working through issues and setting priorities. We really need more people dedicated to Arctos as their actual job....

**Comment by Jegelewicz** [comment]
Adding this here because I think it fits. From the Yale talk at SPNHC today

LinkedArt - https://linked.art/

Linked Art is a [Community](https://linked.art/community/) working together to create a shared [Model](https://linked.art/model/) based on [Linked Open Usable Data](https://linked.art/loud/) to describe cultural heritage with a particular focus on art. We specify [APIs](https://linked.art/api/) that enable access to data in the model, and implement them in [Software](https://linked.art/software/) to provide valuable content to the world. It is approaching a first stable release and we welcome additional partners and collaborators.
